src/layers/BitmapLayer.js

```diff
@@ -0,0 +1,98 @@
+import { BitmapLayer as BaseBitmapLayer } from '@deck.gl/layers';
+import { COORDINATE_SYSTEM } from '@deck.gl/core';
+import { Model, Geometry } from '@luma.gl/core';
+import GL from '@luma.gl/constants';
+
+const PHOTOMETRIC_INTERPRETATIONS = {
+  WhiteIsZero: 0,
+  BlackIsZero: 1,
+  RGB: 2,
+  Palette: 3,
+  TransparencyMask: 4,
+  CMYK: 5,
+  YCbCr: 6,
+  CIELab: 8,
+  ICCLab: 9
+};
+
+const defaultProps = {
+  pickable: true,
+  coordinateSystem: COORDINATE_SYSTEM.CARTESIAN,
+  channelData: { type: 'object', value: {}, compare: true },
+  bounds: { type: 'array', value: [0, 0, 1, 1], compare: true },
+  opacity: { type: 'number', value: 1, compare: true }
```

Comment:
same here, does opacity have any effect on this layer?

Reply:
https://deck.gl/docs/api-reference/core/layer#opacity I think it does.